packages/fxa-content-server/app/scripts/lib/mailcheck.js

```diff
@@ -35,6 +35,53 @@ const SECOND_LEVEL_DOMAINS = [
   'outlook',
   'live',
   'yahoo',
+  'icloud',
```

Comment:
Will adding these domains to mailcheck skew the A/B test since all users will receive earlier feedback as to whether a domain might be invalid or not?

Reply:
I think so.  I can add these domains in another PR once the experiment's over.